LAD-AMA-Common/metrics_ext_utils/metrics_ext_handler.py

```diff
@@ -617,9 +617,14 @@ def setup_me(is_lad):
                         data = line.split("=")
                         aad_auth_url = data[1][1:-1] # Removing the quotes from the front and back
                         break
-
-    if aad_auth_url == "":
-        raise Exception("Unable to find AAD Authentication URL in the request error response. Failed to set up ME.")
+    
+    except Exception as e:
+        message = "Failed to retrieve AAD Authentication URL from " + arm_url + " with Exception={0}".format(e)
+        message += "Continuing with metrics setup without AAD auth url."
```

Comment:
nit suggest change to
`... with exception='{0}'. ".format(e)` (note space after period). for clearer formatting etc.